deep_quoridor/src/plugins/wandb_train.py

```diff
@@ -67,8 +69,29 @@ def __init__(
         self.agent_evolution_tournament = agent_evolution_tournament
         self.include_raw_metrics = include_raw_metrics
 
-    def _initialize(self, game):
-        assert self.agent
+        if wandb_run is None:
+            self.run = wandb.init(
+                project=self.params.project,
+                job_type="train",
+                id=self.params.run_id(),
+                group=f"{self.params.run_id()}",
+                notes=self.params.notes,
+            )
+        else:
+            self.run = wandb_run
+        Timer.set_wandb_run(self.run)
+
+        self._initialized = False
+
+    def _initialize(
+        self,
+        game,
+    ):
+        if self._initialized:
```

Comment:
We had been re-initializing every time `start_game()` is called, which seemed unneeded.